src/ResourceManager/Reservations/Commands.Reservations/Cmdlets/GetReservationOrderId.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Azure.Commands.Reservations.Common;
+using Microsoft.Azure.Management.Reservations.Models;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Reservations.Models;
+using Microsoft.Azure.Management.Reservations;
+
+namespace Microsoft.Azure.Commands.Reservations.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmReservationOrderId"), OutputType(typeof(AppliedReservations))]
```

Comment:
This cmdlet name doesn't seem to make sense.  If you are returning applied reservation, why is this not Get-AzureRmAppliedReservation?

Reply:
quote from feedback we got from Cormac:
"Consider adding Get-AzureRmReservationId and changing Get-AzureRmReservation to return the reservations the user has access to in the tenant, as well as getting the reservation they have access to by subscription id"
We are sticking to AzureRmReservationOrderId, this call is a little tricky to define because it is not returning full resource (reservation order) but only reservation order id.